features/step_definitions/implicit_waiting_steps.rb

```diff
@@ -11,21 +9,21 @@
 
 Then('the slow elements are waited for') do
   start_time = Time.now
-  @test_site.slow.even_links(minimum: 1)
+  @test_site.slow.even_links
 
   expect(Time.now - start_time).to be_between(0.15, 0.3)
 end
 
 Then('the slow section is waited for') do
   start_time = Time.now
-  @test_site.slow.first_section(count: 1)
+  @test_site.slow.first_section
 
   expect(Time.now - start_time).to be_between(0.15, 0.3)
 end
 
 Then('the slow sections are waited for') do
   start_time = Time.now
-  @test_site.slow.all_sections(count: 2)
+  @test_site.slow.all_sections
```

Comment:
should we check count here?

Reply:
I suppose there's no harm in re-adding this one back. It was just we needed these in to make it pass on Capybara 2.x

I'll re-add it